testing/kuttl/e2e/pgadmin/02--check-settings.yaml

```
@@ -0,0 +1,26 @@
+---
+apiVersion: kuttl.dev/v1beta1
+kind: TestStep
+commands:
+  # Log the amount of space on the startup volume. Assert that 4KiB are used.
```

```suggestion
  # Log the amount of space on the startup volume. Assert that 4KiB are used.
  # `df` output looks like:
  # ```
  # Filesystem     1K-blocks  Used Available Use% Mounted on
  # tmpfs            2015812     4   2015808   1% /etc/pgadmin
  # ```
```
I'm not crazy about how this ^ looks, but it might be helpful if we ever need to revisit what we're printing. (Maybe?)